src/blocks/Image/ZestyImage.js

```diff
@@ -25,8 +24,15 @@ const ZestyImage = ({
   style = {},
   attributes = {},
 }) => {
-  /* Taking the options object and converting it into a query string. */
+  //  check if exist and assign values to options object
+  if (width) {
+    options.width = width;
+  }
+  if (height) {
+    options.height = height;
+  }
 
```

Comment:
assign the width and height to the options object if exist, in this way I can only pass  width and height from the parent component and will return consistent height and width from CDN and <img/> tag

Reply:
![image](https://user-images.githubusercontent.com/61284357/188871326-70d139ad-1bd9-47f3-b210-2de1ef5165ff.png)


example call from parent

Reply:
HTML rendered 

![image](https://user-images.githubusercontent.com/61284357/188871721-3ee352aa-3c1a-42c2-865c-bd6c44c5b12a.png)

Reply:
Sample call with additional options

![image](https://user-images.githubusercontent.com/61284357/188872001-7426d04f-222e-4a5d-9aad-9fe463b1bcb3.png)

HTML rendered with additional options

![image](https://user-images.githubusercontent.com/61284357/188872118-7e3d1e1c-621d-4450-a0c6-ca64fd0424a6.png)